bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/gpi/Gpi.java

```diff
@@ -320,6 +324,11 @@ public String logValue() {
             return _logValue;
         }
 
+        @Override
+        public boolean isObsolete() {
+            return false;
+        }
+
```

Comment:
This is not in the scope of this PR, but I think at some point we should get rid of all of these default implementations of the `ObsoletableSPType` interface `isObsolete` method that return `false` and just change `ObsoletableSpType` to have:

`default boolean isObsolete() { return false; }`

There are so many of them.
Maybe I'll do this if I have a few minutes.